CONTRIBUTING.md

```diff
@@ -26,3 +26,10 @@
 * If it makes sense, add tests for your code and/or run a performance benchmark
 * Make sure all tests pass (`bundle exec rake`)
 * Create a pull request
+
+## Releasing
+
+* Bump the version in `lib/liquid/version.rb` and merge it on `main`
+* Run `rake release` to create the new version tag and deploy the new version to RubyGems
```

Comment:
```suggestion
```

We should remove `rake release` since we now have an internal tool that automated this step